### PR TITLE
Fix repeated onboarding elements (2862)

### PR DESCRIPTION
**Pull request by steins-code**
Resolve duplicated elements during onboarding as described here: https://jira.itc.sap.com/browse/EXPOSUREAPP-2862

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift [hunk 1]
```diff
@@ -84,13 +84,15 @@ final class OnboardingInfoViewController: UIViewController {
 
 	private var onboardingInfos = OnboardingInfo.testData()
 	private var exposureManagerActivated = false
+	private var pageSetupDone = [OnboardingPageType: Bool]()
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:87]: Wouldn't it be enough to have just a bool? The OnboardingInfoViewController is initialized only with one pageType. And the status "done" is independent of the type.

Comment (steins-code) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift]: done

**Review by marcussc — APPROVED** [review]

**Comment by marcussc** [comment]
Please don't forget to set the fix version in Jira, thanks 👍

**Review by melloskitten — APPROVED** [review]
LGTM 💯

**Pull request merged**